pkg/agent/openflow/pipeline.go

```diff
@@ -452,6 +452,14 @@ func (c *client) defaultDropFlow(tableID binding.TableIDType, matchKey int, matc
 		Action().Drop().Done()
 }
 
+// localProbeFlow generates the flow to resubmit packets to l2ForwardingOutTable. The packets are sent from node to probe the liveness of local pods.
```

Comment:
```suggestion
// localProbeFlow generates the flow to resubmit packets to l2ForwardingOutTable. The packets are sent from Node to probe the liveness/readiness of local Pods.
```

Reply:
Thanks for suggestions, I will apply it.